bin/pre-commit

```diff
@@ -2,8 +2,9 @@
 
 require 'pre-commit/cli'
 
-if !File.exists?(".git")
-  abort "No .git directory found."
-end
+# Change directory to the root of the git repository.
+repo_root = `git rev-parse --show-toplevel`.strip
+abort "No .git directory found." unless File.directory?(repo_root)
```

Comment:
Should this be:

```ruby
unless File.directory?(File.join(repo_root, ".git"))
```

Or am I misunderstanding how `rev-parse --show-toplevel` works?